src/Steam.Common/AccountType.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Steam
```

Comment:
Shouldn't this and the other namespaces be `Steam.Common`?

Reply:
Was your stuff in Steam.Common?

Reply:
Yeah it is, feel free to change it.

Reply:
Nope, I checked, it's in `Steam` (I set the root namespace ahead of time).